src/ui/views/controller.coffee

```diff
@@ -42,68 +42,6 @@ handle 'update:viewstate', ->
     setLeftSize viewstate.leftSize
     setConvMin viewstate.showConvMin
     if viewstate.state == viewstate.STATE_STARTUP
-        if Array.isArray viewstate.size
```

Comment:
How does this work when you close Yakyak on a 2 monitor setup and then open it again on a 1 monitor setup?

I can't test this scenario right now.

This code was put in place because of that *(see the comments)*

Reply:
Just tested this, and it opens just fine at the right side on my primary monitor if I move it to my secondary (it's to the right of my primary), close it, and disconnect the secondary monitor. So it looks like this is working at least.